### PR TITLE
Prepare spring nov release

### PR DESCRIPTION
**Pull request by saragluna**
As title. Update Spring library versions.

### CODE REVIEW

File: sdk/spring/azure-identity-spring/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
+## 1.11.0 (2021-11-24)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Regular updates for dependency versions.
```
Comment (chenrujun) [inline sdk/spring/azure-identity-spring/CHANGELOG.md:5]: 1. What does `Regular updates for dependency versions.` mean? Does it mean `### Dependency Upgrades`? We didn't update spring-boot version. Maybe you mean Azure SDK version?
2. Is `This release is compatible with Spring Boot 2.5.0 - 2.5.4.` is need here?

Comment (saragluna) [reply sdk/spring/azure-identity-spring/CHANGELOG.md:5]: Yes, it means the Azure SDK version.
Sure, I will add the compatibility infomation.

Comment (chenrujun) [reply sdk/spring/azure-identity-spring/CHANGELOG.md:5]: > Yes, it means the Azure SDK version.

How about:
1.  Move this into `### Dependency Upgrades` section? 
2. And list these versions? (Maybe not necessary for now.)

Comment (saragluna) [reply sdk/spring/azure-identity-spring/CHANGELOG.md]: I prefer not to, the bom will include the versions.

File: sdk/spring/azure-identity-spring/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
+## 1.11.0 (2021-11-24)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Regular updates for Azure SDK dependency versions. This release is compatible with Spring Boot 2.5.0 - 2.5.4.
```
Comment (chenrujun) [inline sdk/spring/azure-identity-spring/CHANGELOG.md:5]: To keep the same style of all changelog entries, I prefer write like this:

```
## 1.11.0 (2021-11-24)
This release is compatible with Spring Boot 2.5.0 - 2.5.4.

### Dependency Upgrades
- Regular updates for Azure SDK dependency versions.

```

Comment (saragluna) [reply sdk/spring/azure-identity-spring/CHANGELOG.md:5]: fair enough

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**